frontend/src/features/admin-form/settings/components/AuthSettingsSection/EsrvcIdBox.tsx

```diff
@@ -65,9 +66,17 @@ export const EsrvcIdBox = ({
       case FormAuthType.SP:
       case FormAuthType.CP:
       case FormAuthType.MyInfo:
-        return 'Contact askNDI@tech.gov.sg for your e-service ID'
+        return (
+          <Text textStyle="body-2" color="secondary.400">
+            Contact{' '}
+            <Link isExternal href="https://go.gov.sg/formsg-singpass-contact">
+              Singpass partner support
+            </Link>{' '}
+            for your e-Service ID
+          </Text>
```

Comment:
Is there a reason for memoizing the entire component instead of just the string?

Reply:
yeah, makes sense. based on react docs, useMemo might reduce the re-rendering of this component if the settings.AuthType is the same https://react.dev/reference/react/useMemo#skipping-re-rendering-of-components

however, since this does not seem to be an expensive component to render, removing useMemo seems to be the right call here

Reply:
This function now looks like a fully fledged component. May I suggest that we make it a component i.e., `<HelperText authType={authType} />`?

Previously, it was only responsible for determining the string to render, however, now it had obtained more responsibility -- to also dictate how the string renders.

Reply:
resolved! i have moved it to its own component